sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -78,6 +78,8 @@ final class ManifestMetadataReader {
   static final String CLIENT_REPORTS_ENABLE = "io.sentry.send-client-reports";
   static final String COLLECT_ADDITIONAL_CONTEXT = "io.sentry.additional-context";
 
+  static final String PERFORM_ACTIVITY_FRAMES_TRACKING = "io.sentry.activity-frames-tracking";
```

Comment:
No strong opinions here (feel free to ignore) but it seems there's a theme of having `io.sentry.x.enable` for bool flags. But it also seems it's not consistent.

Also did we want to name this without `activity` in the name so it's more generic?

Reply:
I was told ".enable" is legacy, but yes I renamed it to "frames tracking" now 👍